js/components/closed-caption/textEnhancementsTab.js

```diff
@@ -76,4 +93,6 @@ var TextEnhancementsTab = React.createClass({
   }
 });
 
-module.exports = TextEnhancementsTab;
\ No newline at end of file
+TextEnhancementsTab = AccessibleMenu(TextEnhancementsTab, { useRovingTabindex: true });
+
```

Comment:
Should we define the propTypes for TextEnhancementsTab?

Reply:
Will add them